modular-docs-manual/files/TEMPLATE_PROCEDURE_doing-one-procedure.adoc

```diff
@@ -29,11 +40,23 @@ This paragraph is the procedure module introduction: a short description of the
 
 . Use an unnumbered bullet (*) if the procedure includes only one step.
 
-//.Verification steps
-//(Optional) Provide the user with verification method(s) for the procedure, such as expected output or commands that can be used to check for success or failure.
+.Verification steps
+////
+Optional. Delete if not applicable. Provide the user with verification methods for the procedure, such as expected output or commands that can be used to check for success or failure.
+////
 
-.Additional resources
+. Start each step with an active verb.
 
+. Include one command or action per step.
```

Comment:
I would advocate for one "non-trivial" command per step. For example, if your procedure is for some kind of GUI form or wizard and users click a "Next" button to proceed to the next step, that doesn't need to be a separate step. 

Compare the difference:

. Do this thing.
. Select "Next".
. Do this other thing.
. Select "Next".

VS 

. Do this thing and then select "Next".
. Do this other thing and then select "Next".